client/app/components/edit-in-place.js

```diff
@@ -1,6 +0,0 @@
-import { react2angular } from 'react2angular';
```

Comment:
I don't see why this file was needed, since it automatically turns the React component into the edit-in-place Angular component. @washort Can you remember why you had both edit-in-place.js and EditInPlace.jsx?